### ISSUE TITLE
Backwards Docking should play movement animation in reverse

### ISSUE THREAD

**Issue by IceReaper** [IceReaper/KKnD#4]

**Comment by IceReaper** [comment]
The problem is that the actor is moved via Drag. However this triggers the Mobile code so swap the animation to "Move".
There is no detection if its forward/backward. Also there is no PlayRepeatingBackwards.

**Comment by Dzierzan** [comment]
Would adding additional sequence for actors help, though?

For example:
	
move-backward: sprites.lvl|Mastodon.mobd
	Start: 48
	Length: *
        Frames: 58,57,56,55,54,53,52,51,50,49,48
	Tick: 120
	Facings: -16

**Comment by IceReaper** [comment]
No, the standard actor movement code detects movement and sets the animation to "Move". that code must be enhanced in openra to detect backwards movement and then play the animation reversed.
